app-cdr/ecm/ecm-1.0.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit toolchain-funcs versionator
+
+MY_PV=$(delete_all_version_separators)
+
+DESCRIPTION="CD EDC/ECC code (un)stripper"
+HOMEPAGE="https://web.archive.org/web/20091104002036/www.neillcorlett.com/${PN}"
+SRC_URI="https://web.archive.org/web/20091021035854/www.neillcorlett.com/downloads/${PN}${MY_PV}.zip"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~arm ~mips ~ppc ~ppc64 ~amd64-linux ~x86-linux ~x64-macos"
+
+DEPEND="app-arch/unzip"
+
+S="${WORKDIR}"
+
+src_compile() {
+	for i in *.c; do
```

Comment:
Not a good solution. This makes parallel make inefficient, and as a result breaks e.g. distcc. I get this package probably ain't very large but still not a nice way of doing it. You can probably play with inline make, e.g. `emake -f - <<-EOF` (as a cheap alternative to adding files) ;-).